feature/movie-detail/src/main/java/io/filmtime/feature/movie/detail/MovieDetailScreen.kt

```diff
@@ -53,8 +60,10 @@ fun MovieDetailScreen(
     CircularProgressIndicator(
       modifier = Modifier.wrapContentSize(),
     )
-  } else if (state.message != null) {
-    Text(text = state.message!!)
+  } else if (state.error != null) {
+    ShowError(state.error!!, state.message!!) {
+      viewModel.load()
+    }
```

Comment:
I guess this one is more readable. WDYT?
```suggestion
    ShowError(
       error = state.error!!,
       message = state.message!!,
       onRefresh = viewModel::load
     )
```